### PR TITLE
Add option to always start PolyMC fullscreen.

### LINKED ISSUE #1176: Add command line option and/or setting to open PolyMC in a maximized window
### Role

Opening an instance of PolyMC through Gaming Mode on Steam Deck.

### Suggestion

I want to be able to open PolyMC and/or the log window (when opening a specific instance through the command line)

### Benefit

This could be useful for use on Steam Deck, since there's no way to maximize a window in Gaming Mode. I'd like to have both the launcher and the log window to fill up the entire screen on my Deck when I open PolyMC.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

_No response_

### PR DESCRIPTION
**Pull request by slendidev**
Closes #1176.

### CODE REVIEW

File: launcher/ui/pages/global/LauncherPage.ui [hunk 1]
```diff
@@ -380,6 +380,13 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="startFullscreenCheckBox">
+            <property name="text">
+             <string>Start fullscreen</string>
```
Comment (ghost) [inline launcher/ui/pages/global/LauncherPage.ui]: Probably capitalize the F

File: launcher/ui/pages/global/LauncherPage.ui [hunk 2]
```diff
@@ -38,7 +38,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```
Comment (ghost) [inline launcher/ui/pages/global/LauncherPage.ui]: Common Qt Designer L

**Review by ghost — CHANGES_REQUESTED** [review]

File: launcher/ui/pages/global/LauncherPage.ui (hunk 2 above)
Comment (slendidev) [reply launcher/ui/pages/global/LauncherPage.ui]: I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER I HATE QT DESIGNER

**Comment by LennyMcLennington** [comment]
What's the benefit of starting in fullscreen mode on top of everything? PolyMC already remembers the window geometry you closed with.

**Comment by LennyMcLennington** [comment]
@xslendix I looked at the issue that this apparently closes, and it's asking for 'start maximised' not 'start fullscreen'.

**Comment by slendidev** [comment]
This PR is not relevant to the issues it's trying to fix and I do not think it's even remotely useful. Closing, might make a PR that actually addresses the issue raised.

**Pull request closed without merging**